types/scryptsy/scryptsy-tests.ts

```diff
@@ -0,0 +1,14 @@
+import * as scrypt from 'scryptsy';
```

Comment:
Since `scrypt` is function, it should be `import ... = require` rather than ES module